src/main/java/br/unb/cic/analysis/df/DataFlowAbstraction.java

```diff
@@ -19,35 +14,35 @@
 public class DataFlowAbstraction {
 
     private Local local;
-    private JInstanceFieldRef localInstanceField;
-    private StaticFieldRef localStaticFieldRef;
+    private JInstanceFieldRef localField;
+    private StaticFieldRef localArrayFieldRef;
```

Comment:
@galilasmb and @barbosamaatheus does this make sense? what are you trying to represent here? that a static field was assigned by someone? or that a variable of type array was assigned by someone?

Reply:
It is a representation for a static arrays.

Reply:
@galilasmb so we only deal with static variables that are arrays? and we just deal with instance variables that are not arrays? or the difference is not on whether a variable is an array or nor, and just if it's instance or static, and you should drop "Array" (and maybe add static) from the name?

Reply:
Soot generates different names when an array is static, so we are dealing with both, when it is and when it is not. When is static, it divides in many units similar to the Fields analysis with the word and chain  "$stack".